starter-code/routes/index.js

```diff
@@ -1,9 +1,15 @@
 const express = require('express');
 const router  = express.Router();
 
+const Celebrity = require ("../models/celebrity") //los models siempre con mayuscula 
```

Comment:
You are not using the celebrity model here in this file, so there is no need for this line here